tests/Driver/PhpAmqp/DriverTest.php

```diff
@@ -1,26 +1,27 @@
 <?php
 
-namespace Bernard\Tests\Driver;
+namespace Bernard\Tests\Driver\PhpAmqp;
 
-use Bernard\Driver\PhpAmqpDriver;
+use Bernard\Driver\PhpAmqp\Driver;
 use PhpAmqpLib\Channel\AMQPChannel;
 use PhpAmqpLib\Connection\AMQPStreamConnection;
 use PhpAmqpLib\Message\AMQPMessage;
+use PHPUnit\Framework\TestCase;
 
-class PhpAmqpDriverTest extends \PHPUnit\Framework\TestCase
+final class DriverTest extends TestCase
 {
     /**
-     * @var AMQPStreamConnection
+     * @var AMQPStreamConnection|\PHPUnit_Framework_MockObject_MockObject
```

Comment:
Isn't this still the legacy object name? As of 6.0 there should be properly namespaced class names.

Reply:
Namespace support for the mockobject lib was just merged sebastianbergmann/phpunit-mock-objects#376. So in a next version we will be able to replace this!